### ISSUE TITLE
Unable to retrieve my url with file extension -> Files.link

### ISSUE THREAD

**Issue by Antony-Eliga** [VeliovGroup/Meteor-Files#552]
### I'm having an issue:
  - Files.link(fileRef) -> I am unable to retrieve my url with file extension (works with version <= 1.9.2)
  - Meteor-Files: Version 1.9.3
  - Meteor Version: 1.6.0.1
  - Appearing on Mac/Linux
  - Server Side issue

Maybe caused by 
``` 
if (vRef.extension && _.isString(vRef, 'extension'))
```
One correction may be
```
if (vRef.extension && _.isString(vRef.extension))
```

**Comment by dr-dimitru** [comment]
Hello @Antony-Eliga ,

Thank you for reporting about this.
Fix is on the way.

**Comment by dr-dimitru** [comment]
@Antony-Eliga thank you for reporting about this.
Published as [`v1.9.6`](https://github.com/VeliovGroup/Meteor-Files/releases/tag/1.9.6). 

Feel free to reopen it in case if the issue is still persists on your end.
